app/views/home/subscriptions.html.erb

```diff
@@ -71,7 +71,26 @@
           <a rel="tooltip" title="Once per day" class="btn btn-default btn-xs" href="javascript:void(0)"><i class="fa fa-list"></i> Digest</a>
           -->
           <script> input_<%= subscription.tid %> = '#share-<%= subscription.tid %>'</script>
-          <a class="btn btn-outline-secondary btn-sm text-body" rel="popover"  data-toggle="popover" data-placement="left" title="<%= translation('home.subscriptions.share_subscription_link',{},false) %>" data-html="true" data-content="<div style='margin-top:10px;'><a style='padding-left:3px;' href='https://twitter.com/intent/tweet?text=🎈 Public Lab: Subscriptions&url=https://publiclab.org/subscriptions/tag/<%= subscription.tagname %>' class='twitter-share-button'>Tweet</a></div> <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src='//platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document,'script','twitter-wjs');</script>
+          <a class="btn btn-outline-secondary btn-sm text-body" rel="popover"  data-toggle="popover" data-placement="left" title="<%= translation('home.subscriptions.share_subscription_link',{},false) %>" data-html="true" data-content="<div style='margin-top:10px;'><a style='padding-left:3px;' href='https://twitter.com/intent/tweet?text=🎈 Public Lab: Subscriptions&url=https://publiclab.org/subscriptions/tag/<%= subscription.tagname %>' class='twitter-share-button'>Tweet</a></div>
+          <script>
+          window.twttr = (function(d, s, id) 
+          { var js, fjs = d.getElementsByTagName(s)[0],
+            t = window.twttr || {};
+            console.log(t);
+            js = d.createElement(s);
+            js.id = id;
+            js.src = 'https://platform.twitter.com/widgets.js';
+            fjs.parentNode.insertBefore(js, fjs);
+
+            t._e = [];
+            t.ready = function(f) {
+              t._e.push(f);
+            };
+
+            return t;
+          }(document, 'script', 'twitter-wjs'));
+          </script> 
+          ">
```

Comment:
@sakshi-2412 can you indent the content between the script tags? Thanks

Reply:
As in, club them in one line?

Reply:
No I mean the content inside the script tag should be 1 tab/2 spaces inside the script tag.

Reply:
Okay, I'll do that, thanks!

Reply:
Hi @sakshi-2412, you haven't indented the content between the script tags.

Reply:
Oh I misunderstood you

![image](https://user-images.githubusercontent.com/85152262/150933030-374f57cc-0d74-48ed-82af-9c7ac2cb12f7.png)

Is this how it should be? (sorry for bothering you so much 😓)

Reply:
You need to reduce the indentation by one tab. The content inside `window.twittr` is fine but the `window.twittr` itself needs an adjustment.

Reply:
Ohh okay..i hope this is fine 
![image](https://user-images.githubusercontent.com/85152262/150952243-62f06feb-1962-40b4-b2d2-d126dea6a79b.png)

Reply:
That's perfect @sakshi-2412!